examples/text_classification/bert/bert_handler.py

```diff
@@ -22,6 +25,13 @@ def __init__(self):
         self.mapping = None
         self.device = None
         self.initialized = False
+        self.name_index = {
```

Comment:
@HamidShojanazeri Is this needed here, given that the list is coming from the index_to_name.json file